Notify/Notify/Notify.Android/Notify.Android.csproj

```diff
@@ -105,6 +105,7 @@
     <AndroidResource Include="Resources\drawable\Notify.png" />
     <AndroidResource Include="Resources\drawable\NotifyMain.png" />
     <AndroidResource Include="Resources\drawable\NotifyMainFade.png" />
+    <AndroidResource Include="Resources\drawable\NotifyNew.png" />
```

Comment:
Rename the new picture with current image name, and add to the current picture "Old" prefix.